go/arrow/array/binarybuilder_test.go

```diff
@@ -50,11 +51,16 @@ func TestBinaryBuilder(t *testing.T) {
 		}
 		assert.Equal(t, v, ab.Value(i), "unexpected BinaryArrayBuilder.Value(%d)", i)
 	}
+	// Zm9v is foo in base64
+	assert.NoError(t, ab.AppendValueFromString("Zm9v"))
 
 	ar := ab.NewBinaryArray()
+	assert.Equal(t, "Zm9v", ar.ValueString(5))
+
 	ab.Release()
 	ar.Release()
 
+
```

Comment:
Sure. BTW - maybe we should open an issue on adding a linter that runs fmt on all files as a pre-set ? this way it wont need review as I'll be able to just see in the CI. WDYT ? :)

Reply:
I think that it would definitely be a good idea to add that to the existing linting. Right now the linter just runs `staticcheck`, but doesn't check the formatting.